desktop/apps/article/stylesheets/index.styl

```diff
@@ -2,6 +2,7 @@
 @require '../../../components/articles_grid/stylesheets'
 @require '../../../components/tooltips'
 @require '../../../components/channel_nav'
+@require '../../articles/stylesheets/venice_banner'
```

Comment:
Noticing the repeated `closeVeniceBanner` methods in this PR and the requiring backward into another app, I think this is indicating `venice_banner` should be moved to the ./desktop/components folder where you can co-locate the stylus, jade template, and a tiny Backbone view that basically looks [like this view](https://github.com/artsy/force/pull/1403/files#diff-803cd0894b2ea6f609c14ab2c45e4d6b) you wrote. That way you can just require the component across apps and not have to repeat yourself. Let me know if that doesn't make sense and happy to explain more IRL.